src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -854,22 +956,22 @@ public void UpdatePackageStatus(IEnumerable<PackageCollectionItem> installedPack
                 .PostOnFailure(nameof(PackageItemViewModel), nameof(ReloadPackageVersionsAsync));
 
             NuGetUIThreadHelper.JoinableTaskFactory
-                .RunAsync(ReloadPackageMetadataAsync)
-                .PostOnFailure(nameof(PackageItemViewModel), nameof(ReloadPackageMetadataAsync));
+                .RunAsync(ReloadTopLevelPackageMetadataAsync)
+                .PostOnFailure(nameof(PackageItemViewModel), nameof(ReloadTopLevelPackageMetadataAsync));
 
             OnPropertyChanged(nameof(Status));
         }
 
         public void UpdateTransitivePackageStatus(NuGetVersion installedVersion)
```

Comment:
Would it make sense for the PackageItemViewModel to keep track of whether or not the package is transitive?

Reply:
It does in PackageLevel. I know, this method should not be public and we should instead call UpdatePackageStatus and the ViewModel should do the right thing depending on whether it's top-level or transitive. However, this code existed already and at this point I want to make as few changes as possible so we can complete this work and make that enhancement later.

Reply:
Sounds good to me!